content/en/blog/_posts/2022-10-03-current-state-2019-third-party-audit.md

```diff
@@ -0,0 +1,116 @@
+
+{{</note>}}
+
+| Title                                                                            | Issue                                                           | Time Commitment | Complexity | Benefit to Ecosystem |
+|----------------------------------------------------------------------------------|-----------------------------------------------------------------|-----------------|------------|----------------------|
+| Kubernetes does not facilitate certificate revocation                            | [#81111](https://github.com/kubernetes/kubernetes/issues/81111) | High            | High       | Medium               |
+| Use of InsecureSkipVerify and other TLS weaknesses                               | [#81119](https://github.com/kubernetes/kubernetes/issues/81119) | High            | High       | Medium               |
+| Kubectl can cause a local Out Of Memory error with a malicious Pod specification | [#81123](https://github.com/kubernetes/kubernetes/issues/81123) | Medium          | Medium     | Medium               |
+| Improper fetching of PIDs allows incorrect cgroup movement                       | [#81124](https://github.com/kubernetes/kubernetes/issues/81124) | Medium          | Medium     | Medium               |
+| kubelet liveness probes can be used to enumerate host network                    | [#81129](https://github.com/kubernetes/kubernetes/issues/81129) | High            | High       | Medium               |
+| TOCTOU when moving PID to manager's cgroup via kubelet                           | [#81113](https://github.com/kubernetes/kubernetes/issues/81113) | Medium          | Medium     | Low                  |
+| Log rotation is not atomic                                                       | [#81132](https://github.com/kubernetes/kubernetes/issues/81132) | Medium          | Medium     | Low                  |
+| Arbitrary file paths without bounding                                            | [#81133](https://github.com/kubernetes/kubernetes/issues/81133) | Medium          | Medium     | Low                  |
+| Services use questionable default functions                                      | [#81138](https://github.com/kubernetes/kubernetes/issues/81138) | Medium          | Medium     | Low                  |
+| Use standard formats everywhere                                                  | [#81140](https://github.com/kubernetes/kubernetes/issues/81140) | High            | High       | Very Low             |
+| Hardcoded use of insecure gRPC transport                                         | [#81142](https://github.com/kubernetes/kubernetes/issues/81142) | High            | High       | Very Low             |
+
+To get started on fixing any of these findings that need help, please
+consider getting involved in [Kubernetes SIG
+Security](https://github.com/kubernetes/community/tree/master/sig-security#contact)
+by joining our bi-weekly meetings or hanging out with us on our Slack
+Channel.
```

Comment:
Do we want to put a link to the K8s Slack or alternately specify #sig-security as the channel?
```suggestion
Channel.
```

Reply:
It's part of the linked SIG Contact section. So I think okay to skip, IMO.